scenarios/container_etcd.yml

```diff
@@ -0,0 +1,6 @@
+scenarios:
```

Comment:
We might want to add a way to filter the number of pods to pick to kill the container. For example, killing the container in one of the etcd pods vs all 3 pods. Thoughts?

Reply:
Updated, please take a look. Let me know if you can think of a better way to get the random pods and not keep killing the same one.